integration-tests/esbuild/basic-test.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const tracer = require('../../').init() // dd-trace
+
+const assert = require('assert')
+const express = require('express')
+const http = require('http')
+
+const app = express()
+const PORT = 31415
+
+assert.equal(express.static.mime.types.ogg, 'audio/ogg')
+
+const server = app.listen(PORT, () => {
+  setImmediate(() => {
+    http.request(`http://localhost:${PORT}`).end() // query to self
+  })
+})
+
+app.get('/', async (_req, res) => {
+  assert.equal(
+    tracer?.scope()?.active()?.context()?._tags?.component,
```

Comment:
Optional chaining doesn't work in 12.x so this can't be backported.

Reply:
This has been removed